Swap 'weight' and 'activation' observers in QConfig.

ZolotukhinM opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#25929 Swap 'weight' and 'activation' observers in QConfig.**

Pull Request resolved: https://github.com/pytorch/pytorch/pull/25929

Differential Revision: [D17288317](https://our.internmc.facebook.com/intern/diff/D17288317)

Commit:
Weight and activation observers are swapped in insertObservers. Fix it.

The actual order is specified in torch/quantization/QConfig.py and is
['weight', 'activation'].

jerryzh168 reviewed (commented):
oh, I feel it's better to put activation as the first argument actually, maybe we should fix https://github.com/pytorch/pytorch/blob/master/torch/quantization/QConfig.py instead?

Commit:
Update on "Swap 'weight' and 'activation' observers in QConfig."


Pull Request resolved: https://github.com/pytorch/pytorch/pull/25929

Differential Revision: [D17288317](https://our.internmc.facebook.com/intern/diff/D17288317)

jerryzh168 commented:
@ZolotukhinM I submitted a PR to change this: https://github.com/pytorch/pytorch/pull/25950

ZolotukhinM commented:
Closing in favor of #25950.
